common.py

```diff
@@ -71,18 +87,20 @@ class PodInfo:
 
 @dataclass
 class TestMetadata:
-    test_case_id: TestCaseType
-    test_type: TestType
     reverse: bool
-    server: PodInfo
-    client: PodInfo
+    test_case_id: TestCaseType = field(default_factory=enum_factory(TestCaseType))
```

Comment:
does this actually work? `default_factory` should be a callable without any argument. But `enum_factory()` returns a callable that takes a `value:Any` argument.

Reply:
It doesn't work, I remove it later